libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -776,6 +818,9 @@ method publish*(g: GossipSub, topic: string, data: seq[byte]): Future[int] {.asy
 
   g.mcache.put(msgId, msg)
 
+  trace "publish message to peers",
```

Comment:
You said "we can't have them in TRACE as running at that level would pollute too much our simulations' logs", but here it is `trace`. Could you please elaborate?